next/pages/404.tsx

```diff
@@ -20,7 +20,7 @@ const NotFoundPage = () => {
   const { t } = useTranslation('common')
 
   return (
-    <div className="flex h-screen w-screen px-7 py-10 md:pl-36 md:pr-32 xl:pl-80 xl:pr-66">
+    <div className="xl:pr-66 flex h-screen w-screen px-7 py-10 md:pl-36 md:pr-32 xl:pl-80">
```

Comment:
Suspitious order - `xl:pl-80` stays at the end, `xl:pr-66` is moved to the start. Is `...-66` spacing defined?

Reply:
You don't have to define custom spacings, etc. anymore in V4.